lib/webserver/WebServer.js

```diff
@@ -162,6 +163,69 @@ const WebServer = function (options) {
         });
     });
 
+    // Telegram settings
+    this.app.get("/api/get_telegram_config", function (req, res) {
+        if(self.configuration.get("telegram_token")){
+            token = self.configuration.get("telegram_token");
+            password = self.configuration.get("telegram_password");
+            clients = self.configuration.get("telegram_clients");
+            telegram_send_consumables = self.configuration.get("telegram_send_consumables");
+            telegram_send_consumables_every = self.configuration.get("telegram_send_consumables_every");
```

Comment:
Can't see where all these variables were declared before use.

Reply:
Yes it's my first nodejs program ;). I will make it better